mediorum/server/serve_blob.go

```diff
@@ -59,6 +64,43 @@ func (ss *MediorumServer) getBlob(c echo.Context) error {
 			return err
 		}
 		defer blob.Close()
+
+		if blob.ContentType() == "audio/mpeg" {
+			fileSize := blob.Size()
+			contentRange := c.Request().Header.Get("Range")
+			if contentRange == "" {
+				contentRange = "bytes=0-"
+			}
+
+			// Calculate ranges for streaming partial content
```

Comment:
would be good to see some small unit tests for this code. we have had a number of bugs in the past from "off by 1" kind of things in this realm